packages/client/tests/functional/issues/15084-mongo-logging/tests.ts

```diff
@@ -0,0 +1,52 @@
+import { NewPrismaClient } from '../../_utils/types'
+import testMatrix from './_matrix'
+// @ts-ignore
+import type { PrismaClient } from './node_modules/@prisma/client'
+
+declare let newPrismaClient: NewPrismaClient<typeof PrismaClient>
+
+testMatrix.setupTestSuite(
+  () => {
+    let client: PrismaClient
+
+    afterAll(async () => {
+      await client.$disconnect()
+    })
+
+    test('should log queries', async () => {
+      client = newPrismaClient({
+        log: [
+          {
+            emit: 'event',
+            level: 'query',
+          },
+        ],
+      })
+
+      const queryLogPromise = ((): Promise<any> =>
+        new Promise((resolve) => {
+          // @ts-ignore
```

Comment:
```suggestion
          // @ts-expect-error
```

`@ts-ignore` gets stripped before type tests